compass/landice/tests/mismipplus/tasks.py

```diff
@@ -0,0 +1,114 @@
+import xarray as xr
+
+# Following from:
+#   compass/ocean/tests/global_ocean/tasks.py
+
+# number of cells at 1000m resolution (with no gutter present). This is used
+# as heuristic to scale the number of cells with resolution, in order to
+# constrain the resoucres (i.e. `ntasks`) based on the mesh size.
+ncells_at_1km_res = 61382
+
+
+def get_ntasks_from_cell_count(config, cell_count):
+    """
+    Computes `ntasks` and `min_tasks` based on mesh size (i.e. cell_count)
+
+    Parameters
+    ----------
+    config : compass.config.CompassConfigParser
+        Configuration options for the test case the step belongs to
+
+    cell_count : int
+        Number of horizontal cells in the mesh. The value of this parameter is
+        caclulated (or approximated) using the functions below.
+
+    Returns
+    -------
+    ntasks : int
+        The target number of MPI tasks
+
+    min_tasks : int
+        The minimum number of tasks
+    """
+
+    # read MPI related parameters from configuration file
```

Comment:
Can we just use `config.getint()` instead of getting floats and then converting to floats below?

Reply:
I've switched to parsing all the parameters as `int`s and then casting them as floats where need be. This was done as of [f916f60](https://github.com/MPAS-Dev/compass/pull/737/commits/f916f60ccc9d7b8e515ef6a8a775fd71979c5b40). Thanks for catching this!

Reply:
Do we need to cast these as floats at all? It does no harm, but it will give exactly the same answer as using ints.

Reply:
Your right, in python casting as floats is kind of overkill. Especially since all of the floating point operations with `int`s we are doing are division, where we are explicitly using the floating point operator (e.g. `/` vs. `//`). 

I was just trying to follow best practice, but seems a little unnecessary here and at the expense of readable code. I'll remove the casting as floats to make things a little more concise.

Reply:
Thanks. I wasn't sure what was considered best practice here, but I think it is more readable without casting them as floats.